<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubWindow</class>
 <widget class="QDialog" name="SubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>500</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>451</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>741</width>
     <height>451</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_7">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="button00">
       <property name="text">
        <string>重庆</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="button01">
       <property name="text">
        <string>北京</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="button02">
       <property name="text">
        <string>香港</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="button03">
       <property name="text">
        <string>上海</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="button04">
       <property name="text">
        <string>澳门</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="button05">
       <property name="text">
        <string>天津</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="button10">
       <property name="text">
        <string>兰州</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="button11">
       <property name="text">
        <string>白银</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="button12">
       <property name="text">
        <string>敦煌</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="button20">
       <property name="text">
        <string>石家庄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="button21">
       <property name="text">
        <string>唐山</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="button22">
       <property name="text">
        <string>张家口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>郑州</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>开封</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>洛阳</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>济南</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="text">
        <string>青岛</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="text">
        <string>烟台</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>日照</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_20">
       <property name="text">
        <string>南昌</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_21">
       <property name="text">
        <string>九江</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBox_22">
       <property name="text">
        <string>上饶</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkBox_23">
       <property name="text">
        <string>赣州</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_6">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_24">
       <property name="text">
        <string>成都</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_25">
       <property name="text">
        <string>绵阳</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBox_27">
       <property name="text">
        <string>自贡</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkBox_26">
       <property name="text">
        <string>泸州</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>531</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>142</width>
      <height>650</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>热门</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>甘肃</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>河北</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>河南</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>山东</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button5">
       <property name="text">
        <string>江西</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>四川</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>江苏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>浙江</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>安徽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>黑龙江</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>吉林</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_33">
       <property name="text">
        <string>辽宁</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_34">
       <property name="text">
        <string>内蒙古</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_35">
       <property name="text">
        <string>新疆</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_36">
       <property name="text">
        <string>西藏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_37">
       <property name="text">
        <string>青海</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_38">
       <property name="text">
        <string>宁夏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_39">
       <property name="text">
        <string>陕西</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_40">
       <property name="text">
        <string>山西</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_41">
       <property name="text">
        <string>湖北</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_42">
       <property name="text">
        <string>湖南</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
